agent_shared/tests/.env.test

```diff
@@ -10,3 +10,6 @@ TEST_CREDENTIAL_LOGO_URL=https://my-domain.example.org/credential_logo.png
 TEST_STRONGHOLD_PATH="../agent_secret_manager/tests/res/test.stronghold"
 TEST_STRONGHOLD_PASSWORD="secure_password"
 TEST_ISSUER_KEY_ID="9O66nzWqYYy1LmmiOudOlh2SMIaUWoTS"
+
+# AGENT_VERIFICATION
+TEST_URL=https://my-domain.example.org
```

Comment:
Shouldn't we also add this to `.env.example`? And what's the difference to `AGENT_APPLICATION_URL`?

Reply:
You're right, that it is actually the same as `AGENT_APPLICATION_URL`, which is not accessible in `agent_verification`. Since I have some ideas about how to approach ServerConfig related env variables in general that require some changes, for now I will (temporarily) add a new env variable: `AGENT_VERIFICATION_URL=${AGENT_APPLICATION_URL}`.